MiniApp/Classes/core/API/MetaDataAPI.swift

```diff
@@ -0,0 +1,22 @@
+internal class MetaDataAPI {
+    let environment: Environment
+
+    init(environment: Environment) {
```

Comment:
not really important but maybe `init(with environment: Environment)` is more elegant